pkg/sql/sem/tree/testdata/eval/cast

```diff
@@ -1070,3 +1070,48 @@ eval
 '-10'::interval::decimal
 ----
 -10.000000000
+
+eval
+'1969-12-30-01.00.00'::timestamp
```

Comment:
please put these near the other timezone cases. they do not have to be exhaustive either -- just one or two is fine (these should be unit tests instead if we need them)